src/platforms/kotlin-multiplatform/swift-package-manager.mdx

```diff
@@ -1,11 +1,15 @@
 ---
 title: Swift Package Manager (SPM)
 description: "Learn about installing the Sentry Kotlin Multiplatform SDK with Swift Package Manager."
+sidebar_order: 2
 ---
 
+Swift Package Manager is a powerful tool for managing dependencies in Swift that allows developers to enjoy a more native integration experience with Xcode.
+If you already use Swift Package Manager or prefer it over other package managers on Apple platforms, this guide will show you how to install the Kotlin Multiplatform SDK using Swift Package Manager.
```

Comment:
Should we say that the regular installation method is recommended here and link to the getting started page?